### PR TITLE
LPS-121964 Update journal clay:vertical-card-v2 usages to clay:vertical-card. Highly recommended to update this usages to navigation-card once LPS-123370 is completed

### PR DESCRIPTION
**Pull request by carloslancha**
On the usages where a `user-icon` is being used as sticker in the card you will see this result:

image

As you can see there's no border and the icon color is always black. This will be fixed once a new release of clay lands in portal, so don't worry about it.

Please, take a special look to [the last commit](https://github.com/liferay-echo/liferay-portal/pull/2989/commits/f5e73bdadf5d4d3f813c0067e3e04693f57b7663). As we pointed in the commit message those two usages are candidates to use the new `clay:navigation-card` tag. We highly recommend you to update the usage once https://issues.liferay.com/browse/LPS-123370 is completed.

### CODE REVIEW

**Comment by carloslancha** [comment]
cc @julien @markocikos

File: modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/VerticalCardTag.java [hunk 1]
```diff
@@ -458,10 +458,15 @@ protected int processStartTag() throws Exception {
 
 		StickerTag stickerTag = new StickerTag();
 
+		String stickerCssClass = getStickerCssClass();
 		String stickerIcon = getStickerIcon();
 		String stickerImageSrc = getStickerImageSrc();
 		String stickerLabel = getStickerLabel();
 
+		if (Validator.isNotNull(stickerCssClass)) {
+			stickerTag.setCssClass(stickerCssClass);
```
Comment (julien) [inline modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/VerticalCardTag.java:467]: :bow: thanks for that

File: modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/cards/VerticalCard.js [hunk 2]
```diff
@@ -51,26 +53,36 @@ export default function VerticalCard({
 }) {
 	const [selected, setSelected] = useState(initialSelected);
 
-	const stickerProps = useMemo(
-		() => ({
+	const stickerProps = useMemo(() => {
+		const stickerProps = {
```
Comment (julien) [inline modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/cards/VerticalCard.js:57]: nice

**Review by julien — APPROVED** [review]

**Comment by carloslancha** [comment]
ci:test:sf

**Comment by carloslancha** [comment]
ci:test:relevant

**Comment by carloslancha** [comment]
ci:test:relevant

**Comment by Tim-Cao** [comment]
Just start reviewing

**Comment by Tim-Cao** [comment]
See https://github.com/liferay-echo/liferay-portal/pull/3002

**Comment by Tim-Cao** [comment]
ci:close

**Comment by victorg1991** [comment]
@carloslancha It seems that the check icon is removed from the page when the page is reloaded: (note the blue check 
blue_check

**Comment by victorg1991** [comment]
Note the blue check flickering

**Comment by victorg1991** [comment]
Ups, this doesn't belong to this pr sorry

**Pull request closed without merging**